pkg/network/ebpf/c/co-re/tracer-fentry.c

```diff
@@ -37,20 +37,22 @@ static __always_inline __u32 systemprobe_ino() {
     return (__u32)val;
 }
 
-static __always_inline bool event_in_task(char *prog_name) {
+static __always_inline u64 get_sysprobe_ns_pid_tgid() {
```

Comment:
Can we test if we're getting the correct pids in one of the tracer tests? Just as a sanity check.

Reply:
so the idea here would be to check that a pid coming back from `/connections` was set in the sysprobe ns proc... would that test have to be running in fargate for it to make sense... or could we theoretically force a test to use `tracer-fentry` outside of fargate to check this functionality?

Reply:
The tests do run on one platform for fargate (https://github.com/datadog/datadog-agent/blob/main/pkg/ebpf/ebpftest/buildmode_linux.go#L27), so no need to special case this. A simple test that creates connection and then checks if the pid is as expected would be fine. You have to explicitly opt out of running a particular test on fargate, so you shouldn't have to check if fargate is enabled for this test (it should run and pass for all platforms).